components/brave_wallet_ui/stories/locale.ts

```diff
@@ -880,6 +879,14 @@ provideStrings({
   braveWalletConfirmHidingToken: 'Hide',
   braveWalletCancelHidingToken: 'Cancel',
 
+  // Visible assets modal
+  braveWalletMyAssets: 'My assets',
+  braveWalletAvailableAssets: 'Available assets',
+  braveWalletDidntFindAssetEndOfList: 'Didn\’t find your asset on the list?',
+  braveWalletDidntFindAssetInList:
+    'If you didn\’t find your asset in this list, you can add it manually by using the button below',
```

Comment:
Shouldn't the appostrophe should use the `'` character instead of `’`

Reply:
Fixed!